packages/console/src/pages/SignInExperience/components/Preview/index.tsx

```diff
@@ -29,10 +29,15 @@ function Preview({
   className,
 }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const { data: connectors } = useConnectorGroups();
   const [language, setLanguage] = useState<LanguageTag>('en');
   const [mode, setMode] = useState<Theme>(Theme.Light);
   const [platform, setPlatform] = useState<PreviewPlatform>(PreviewPlatform.DesktopWeb);
   const { languages } = useUiLanguages();
+  const isNativeTabEnabled =
+    connectors?.some((connectorGroup) =>
+      connectorGroup.connectors.some(({ platform }) => platform === ConnectorPlatform.Native)
```

Comment:
Should not show web-only connectors also on the native view?

Reply:
No, when native connectors are not in our configurable options, there is no need to display the native preview.